src/db/index.js

```diff
@@ -6,6 +6,8 @@ import childProcess from 'child_process'
 import mongoose from 'mongoose'
 
 let db = {}
+//I'm not sure why you guys use let 
```

Comment:
We use `let` statements as part of the new ES2015 syntax, more details on `let` vs `var` (and `const`) here: https://strongloop.com/strongblog/es6-variable-declarations/